apps/wallet/src/ui/app/experimentation/feature-gating.ts

```diff
@@ -3,9 +3,13 @@
 
 import { GrowthBook } from '@growthbook/growthbook';
 
-const GROWTHBOOK_API_KEY =
-    process.env.GROWTH_BOOK_API_KEY ?? 'key_dev_dc2872e15e0c5f95';
+import type { JSONValue } from '@growthbook/growthbook';
+import type { WidenPrimitives } from '@growthbook/growthbook/dist/types/growthbook';
```

Comment:
We shouldn't import internal types like this.

Reply:
Agreed, I'll fix this

Reply:
Fix: https://github.com/MystenLabs/sui/pull/5252/files